File: C#/1002 - Área do Círculo.cs
using System;

namespace AreaCirculo{
    class Program{
        static void Main(string[] args){
            double pi = 3.14159, raio, area;

            raio = Convert.ToDouble(Console.ReadLine());
            area = pi * (raio * raio);
            
            Console.WriteLine(string.Format("A={0:0.0000}", area));
        }
    }
}

File: Java/1002 - Área do Círculo.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        double pi = 3.14159, raio, area;
        Scanner valorRaio = new Scanner (System.in);
        raio = valorRaio.nextDouble();
        area = pi * (raio * raio);
        
        System.out.println(String.format("A=%.4f", area));
    } 
}
